### PR TITLE
svt-av1 x86 and multilib

### PR DESCRIPTION
**Pull request by Hello71**
required for libavif, ffmpeg

### CODE REVIEW

**Comment by novomesk** [comment]
Hmm, the [SVT-AV1 project](https://gitlab.com/AOMediaCodec/SVT-AV1) indicate 64bit target only: "_SVT-AV1 Encoder may run on any Windows* or Linux* 64 bit operating systems._"

Isn't it better to suggest the changes as revisions?

**Comment by Hello71** [comment]
arm isn't 64-bit either

File: media-libs/svt-av1/svt-av1-0.8.7.ebuild [hunk 1]
```diff
@@ -34,15 +34,20 @@ src_prepare() {
...
 		-DBUILD_TESTING=OFF
+		-DCMAKE_OUTPUT_DIRECTORY="${BUILD_DIR}"
 	)
 
+	if [[ ${ABI} != amd64 ]]; then
+		mycmakeargs+=( -DCOMPILE_C_ONLY=ON )
+	fi
+
 	cmake_src_configure
 }
```
[the remark this answers is not in the document]
Comment (Hello71) [reply media-libs/svt-av1/svt-av1-0.8.7.ebuild:53]: about the pkg_postinst, i'm not sure it is correct. if the problem is 3 GB memory limit, then won't you have same problem on other 32-bit archs, like arm, ppc, x86? why restrict warning to abi_x86_32?

[the remark this answers is not in the document]
Comment (Hello71) [reply media-libs/svt-av1/svt-av1-0.8.7.ebuild:53]: for the USE mask, i just moved it to package.use.stable.mask instead, temporarily. can request x86 keywording for svt-av1 after merging this. (technically could merge first commit, then request, then do libavif, but this way is faster)

**Comment by telans** [comment]
`media-libs/svt-av1` has now been keyworded for ~86: https://bugs.gentoo.org/823674

**Pull request closed without merging**